### PR TITLE
Add Sonic Boom Shattered Crystal

### PR DESCRIPTION
**Pull request by Estecraft789**
Adds Sonic Boom Shattered Crystal.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/sonic-boom-shattered-crystal/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+region = "Estados Unidos"
+release_date = "2014-11-11"
+
+[[testcases]]
+title = "0004000000127500"
+compatibility = "0"
+date = "2017-09-20"
+version = "HEAD-b4fbb1c"
+cpu = ""
+gpu = ""
```
Comment (Schplee) [inline games/sonic-boom-shattered-crystal/game.dat]: Empty field.

File: games/sonic-boom-shattered-crystal/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+title = "0004000000127500"
+region = "Estados Unidos"
+release_date = "2014-11-11"
+
+[[testcases]]
+title = "0004000000127500"
+compatibility = "0"
+date = "2017-09-20"
+version = "HEAD-b4fbb1c"
+cpu = ""
```
Comment (Schplee) [inline games/sonic-boom-shattered-crystal/game.dat]: Empty field.

File: games/sonic-boom-shattered-crystal/game.dat [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+[[releases]]
+title = "0004000000127500"
+region = "Estados Unidos"
+release_date = "2014-11-11"
+
+[[testcases]]
+title = "0004000000127500"
+compatibility = "0"
+date = "2017-09-20"
+version = "HEAD-b4fbb1c"
```
Comment (Schplee) [inline games/sonic-boom-shattered-crystal/game.dat]: Not the most recent Nightly build.

File: games/sonic-boom-shattered-crystal/game.dat [hunk 4]
```diff
@@ -0,0 +1,19 @@
+title = "Sonic Boom Shattered Crystal"
+description = "Sonic Boom: Shattered Crystal es una aventura independiente que comparte personajes clave, eventos y sabiduría del universo de Sonic Boom con entornos y personajes exclusivos, así como un énfasis en plataformas y puzzles. Además de Sonic, Tails y Knuckles, los jugadores también podrán asumir el papel de Sticks, el nuevo miembro del equipo.\nEditor: SEGA\nDesarrolladora: Sanzura Games"
+needs_system_files = false
+needs_shared_font = false
+
+[[releases]]
+title = "0004000000127500"
+region = "Estados Unidos"
```
Comment (Schplee) [inline games/sonic-boom-shattered-crystal/game.dat]: Refer to the contributions guidelines.

**Comment by TheKoopaKingdom** [comment]
Please fix the issues mentioned above, or this PR will be closed.

**Pull request closed without merging**